src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -491,7 +491,15 @@ bool ColorControlServer::stopMoveStepCommand(app::CommandHandler * commandObj, c
     EndpointId endpoint = commandPath.mEndpointId;
     Status status       = Status::Success;
 
-    if (shouldExecuteIfOff(endpoint, optionsMask, optionsOverride))
+    // StopMoveStep command has no effect on an active color loop.
+    // Fetch if it is supported and active.
+    uint8_t isColorLoopActive = 0;
+    if (ColorControlServer::Instance().HasFeature(endpoint, ColorControlServer::Feature::kColorLoop))
+    {
+        Attributes::ColorLoopActive::Get(endpoint, &isColorLoopActive);
```

Comment:
should we check return codes on these getters?

Reply:
We can, what should we do on failures? return a command failure? What IM error code?

Reply:
I do not actually know ... but at least log and set some random true/false default. Silent ignore seems bad.

Reply:
done